frontend/src/app/jwt-interceptor.service.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { JwtInterceptorService } from './jwt-interceptor.service';
+import { HttpRequest, HttpHandler, HttpEvent } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+
+const fakeObservable = {} as Observable<HttpEvent<any>>;
+
+class FakeHttpHandler extends HttpHandler {
```

Comment:
I used a simple spy object from Jasmine in the Pro Pack, but that works nicely too